src/web/client/common/remoteFetchProvider.ts

```diff
@@ -75,7 +75,7 @@ export async function fetchDataFromDataverseAndUpdateVFS(
         else {
```

Comment:
This condition will only work for English locales. You should probably update your code to detect specific statusCodes first. Or handle specific error types.

You would also likely benefit from using `node-fetch` instead so that you have guaranteed Error-derived exceptions vs strings being thrown. See: [ERROR-HANDLING.md](https://github.com/node-fetch/node-fetch/blob/main/docs/ERROR-HANDLING.md)